File: docs/user/en/use/examples/loading.java
//some custom logic to load proper entity
final SealedEntity entity = session
	.getEntity("Product", 1, associatedDataContent())
	.orElseThrow();
//deserialize the associated data
final ProductStockAvailability stockAvailability = entity.getAssociatedData(
    "stockAvailability", ProductStockAvailability.class,
	new ReflectionLookup(ReflectionCachingBehaviour.NO_CACHE)
);

File: documentation/user/en/use/examples/loading.cs
//some custom logic to load proper entity
ISealedEntity? entity = session
	.GetEntity("Product", 1, AssociatedDataContentAll());
//deserialize the associated data
ProductStockAvailability stockAvailability = entity.GetAssociatedData<ProductStockAvailability>(
    "stockAvailability",
);